index.js

```diff
@@ -184,6 +184,42 @@ var Space = redefine.Class({
     }
   },
 
+  // Space functions
+  /**
+   * Gets all locales enabled in space.
+   * @returns JSON of locale data for space
+   */
+  getLocales: function() {
+    return this.client.request('/spaces/' + this.sys.id + '/locales');
+  },
+
+
+  /**
+   * Creates a locale
+   * @param name  The readable name of the locale, e.g. U.S English
+   * @param code  the i18n abbreviation, e.g. en-US
+   */
+  createLocale: function(name, code) {
```

Comment:
Nope, our api takes care of that. Passing an object also allows to add additional properties in the future.